### ISSUE TITLE
Version Number Mismatch on the macOS Application.

### ISSUE THREAD

**Issue by paul-cossey** [brackets-cont/brackets#299]
### Prerequisites

* [ ] Can you reproduce the problem with `Debug -> Reload Without Extensions`?
* [ ] Did you perform a cursory search to see if your bug or enhancement is already reported?
* [ ] Did you read the [Troubleshooting guide](https://github.com/brackets-cont/brackets/wiki/Troubleshooting)?

For more information on how to write a good bug report read [here](https://github.com/brackets-cont/brackets/wiki/How-to-Report-an-Issue)
For more information on how to contribute read [here](https://github.com/brackets-cont/brackets/blob/master/CONTRIBUTING.md)

### Description

Hi, Folks 

Not really a bug, but I've noticed that there is a version number mismatch on the macOS App. 

The latest release is on the releases page is v2.2.0, but the versions listed in `/Volumes/Brackets\ 2.2.0/Brackets.app/Contents/Info.plist` is `1.14.2` for both `CFBundleVersion` and `CFBundleShortVersionString`

### Steps to Reproduce

`1. Download the App
2. Right Click - Get Info
3. View the version number`

**Expected behavior:** [What you expected to happen]

`The version number in the info.plist should match the release`

**Actual behavior:** [What actually happened]

`The App displays an old version number`

### Versions

Please include the OS and what version of the OS you're running.
Please include the version of Brackets. You can find it under `Help -> About Brackets` (Windows and Linux) or `Brackets -> About Brackets` (macOS)

**Comment by abose** [comment]
Thanks for raising the issue. This happened as the earlier build tooling was from Adobe and we don't have access to Adobes toolig as brackets is now and independent OSS project. So we had to retrofit the old binaries and patch in our changes causing some version number mismatches.

This will be addressed soon as we are nearing completion of native builds of https://phcode.dev , which will be the future foundation of native builds for phcode IDE and Brackets. Stay tuned for updates in the coming months.

**Comment by paul-cossey** [comment]
Thanks for the update @abose much appreciated.

**Comment by abose** [comment]
New versions of phcode(Brackets IDE successor) should be available before December.

Unfortunately the fix is not easy as we dont have access to some proprietary Installer binaries internal to Adobe, and we had to create our own shell. Its nearing completion.

**Comment by gsmc-alx** [comment]
Thanks for getting back to me, and sorry for my impatience.

will there be a Mac desktop version of phcode?

**Comment by abose** [comment]
No worries, it actually helps. We really like to hear the pulse of the community and that usually guides what we invest our resources in. It will be available for Marcos(m1 and Intel binaries), Windows, Linux,  ans browsers(Chrome, Edge et all, Firefox) as well this time.

**Comment by brianlamb** [comment]
Thanks for the report on this, I started going crazy trying to figure out why the release page was downloading the older version. Indeed the About in the app correctly reports 2.2.0 But the App Bundle info in MacOS was confusing the hell out of me.  Thanks for the updates

**Comment by chrisi51** [comment]
> New versions of phcode(Brackets IDE successor) should be available before December.
> 
> Unfortunately the fix is not easy as we dont have access to some proprietary Installer binaries internal to Adobe, and we had to create our own shell. Its nearing completion.

do i understand correctly, that brackets is replaced by https://phcode.dev/ and so only useable via browser?
Or is there any new desktop app, which i don't seem to find?

**Comment by paul-cossey** [comment]
Hey, @abose 

Appreciate you may still have a bunch of heavy lifting to do, but is there any update on this?

Thanks! 😄

**Comment by abose** [comment]
Hello! Beta releases will be available in February. Mostly done, just working on some final issues.

**Comment by paul-cossey** [comment]
Awesome! Thanks for the update. Good luck with the final few issues 🍀

**Comment by paul-cossey** [comment]
Thanks, @abose 

So just to clarify, is Phoenix Code a replacement for Brackets? And Brackets is no longer under active development?

**Comment by abose** [comment]
> Thanks, @abose 
> 
> So just to clarify, is Phoenix Code a replacement for Brackets? And Brackets is no longer under active development?

Yes. Phoenix code is currently supported and in active development. we recommended to migrate to it. Brackets will be back next year as a text editor, stripped down version oh phcode without the code intelligence bits.

**Comment by paul-cossey** [comment]
Gotcha, thanks for clarifying!
